install/helm/open-match/dashboards/README.md

```diff
@@ -0,0 +1,18 @@
+Grafana Dashboards
+==================
+
+Grafana provides a curated view of the Prometheus metrics emitted from
+Open Match. You can add dashboards here by first designing them in Grafana and
+then exporting them via JSON format.
+
+Steps
+-----
+ 1. Sign in to Grafana.
+ 1. Create a dashboard (+ icon > Dashboard)
+    * See https://dzone.com/articles/creating-visualization-in-grafana for
+       details on how to populate a dashboard with charts.
+ 1. Save the dashboard.
+ 1. Click the share icon at the top right.
+ 1. Select "Export"
+ 1. Select "Save to File"
+ 1. Download the file into this directory.
```

Comment:
nit: looks like numbering in off - can you fix that please.

Reply:
Markdown automatically numbers them.

Reply:
Looks like this is how markdowns function - and this renders fine - so resolving this.